build.gradle

```
@@ -9,7 +9,7 @@ buildscript {
     dependencies {
         classpath 'com.android.tools.build:gradle:7.3.1'
         classpath 'com.google.gms:google-services:4.3.14'
-        classpath 'org.jetbrains.kotlin:kotlin-gradle-plugin:1.7.21'
+        classpath 'org.jetbrains.kotlin:kotlin-gradle-plugin:1.8.0-Beta'
```

We only use stable versions of this plugin.
```suggestion
        classpath 'org.jetbrains.kotlin:kotlin-gradle-plugin:1.7.21'
```